spf/dnscache.go

```diff
@@ -0,0 +1,26 @@
+package spf
+
+type dnsCache map[string]map[string][]string
+
+func (c *dnsCache) dnsGet(label, rtype string) ([]string, bool) {
```

Comment:
dnsCache is a map type, so these functions can be declared `func (c dnsCache)` and save some `*`'s in the body.